config/eslintrc.yaml

```diff
@@ -17,6 +17,11 @@ globals:
   mount: true
   snapshotDiff: true
 
+parserOptions:
+  ecmaFeatures:
+    # Remove after migration
+    legacyDecorators: true
```

Comment:
can we remove this?

Reply:
If we do, then our linters would throw warnings for `@decorators` in front of `export` statements (which is deprecated since babel 7). My suggestion would be to add this rule to allow the legacy syntax now and migrate the codebase later when we have time.